domains/mail.icehead.json

```diff
@@ -0,0 +1,16 @@
+{
+  "owner": {
+    "username": "Tanjimfuad",
+    "email": "tanjim_fuad@proton.me"
+  },
+  "record": {
+    "MX": [
+      "mx1.improvmx.com",
+      "mx2.improvmx.com"
+    ]
+  },
```

Comment:
You only need one record key not 2 so please combine the two.

Reply:
I think I fixed it?